### PR TITLE
Improve for issue 1572: func FeasibleNodesToFind to use list with a centain order

### PR DESCRIPTION
**Pull request by lowang-bh**
FIX Issue #1572

### CODE REVIEW

File: pkg/scheduler/cache/cache.go [hunk 1]
```diff
@@ -811,6 +812,7 @@ func (sc *SchedulerCache) Snapshot() *schedulingapi.ClusterInfo {
 		RevocableNodes: make(map[string]*schedulingapi.NodeInfo),
 	}
 
+	snapshot.NodeList = sc.NodeList
```
[the remark this answers is not in the document]
Comment (lowang-bh) [reply pkg/scheduler/cache/cache.go]: oK, fixed it.

**Comment by shinytang6** [comment]
there is something err with the CI, please help check it  @lowang-bh

**Review by Tammy-kunyu — COMMENTED** [review]
cool

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Comment by jasonliu747** [comment]
/hold
@lowang-bh comes up with another elegant solution, will submit that PR on weekend. Let's discuss which solution is more feasible then.

**Comment by jasonliu747** [comment]
@lowang-bh 
image

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**